FluentTerminal.App/Views/SettingsPages/About.xaml

```diff
@@ -45,43 +45,46 @@
+                            -<Hyperlink NavigateUri="{x:Bind ViewModel.CurrentVersionReleaseNotesURL}">Release notes</Hyperlink>
                         </TextBlock>
                     </StackPanel>
 
-                    <StackPanel Margin="0,24,0,0">
+                    <StackPanel Margin="{StaticResource ItemMargin}">
                         <TextBlock Margin="0,0,0,8" Text="Latest version" />
                         <StackPanel Orientation="Horizontal" Visibility="{x:Bind ViewModel.LatestVersionLoading, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}">
-                            <ProgressRing Margin="0,0,8,0" IsActive="True"/>
+                            <ProgressRing Margin="0,0,8,0" IsActive="True" />
                             <TextBlock><Italic>Checking for updates...</Italic></TextBlock>
                         </StackPanel>
                         <TextBlock Visibility="{x:Bind ViewModel.LatestVersionNotFound, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}">
                             <Italic>Not found</Italic>
                         </TextBlock>
                         <TextBlock Visibility="{x:Bind ViewModel.LatestVersionFound, Mode=OneWay, Converter={StaticResource TrueToVisibleConverter}}">
-                            <Run Text="{x:Bind ViewModel.LatestVersion, Mode=OneWay}" /> -
-                            <Hyperlink NavigateUri="{x:Bind ViewModel.LatestVersionReleaseNotesURL, Mode=OneWay}">Release notes</Hyperlink>
+                            <Run Text="{x:Bind ViewModel.LatestVersion, Mode=OneWay}" />
+                            -<Hyperlink NavigateUri="{x:Bind ViewModel.LatestVersionReleaseNotesURL, Mode=OneWay}">Release notes</Hyperlink>
                         </TextBlock>
                     </StackPanel>
 
-                    <StackPanel Margin="0,24,0,0">
+                    <StackPanel Margin="{StaticResource ItemMargin}">
                         <TextBlock Margin="0,0,0,8" Text="License" />
                         <TextBlock><Hyperlink NavigateUri="https://github.com/felixse/FluentTerminal/blob/master/LICENSE">GNU General Public License v3.0</Hyperlink></TextBlock>
                     </StackPanel>
                 </StackPanel>
-                <StackPanel Grid.Row="1" Margin="0,72,0,24" VerticalAlignment="Bottom">
+                <StackPanel
+                    Grid.Row="1"
+                    Margin="0,72,0,0"
+                    VerticalAlignment="Bottom">
                     <TextBlock>
-                        Created by <Italic>felixse</Italic>
-                        and <Hyperlink NavigateUri="https://github.com/felixse/FluentTerminal/graphs/contributors">contributors</Hyperlink>
+                        Created by<Italic>felixse</Italic>
+                        and<Hyperlink NavigateUri="https://github.com/felixse/FluentTerminal/graphs/contributors">contributors</Hyperlink>
```

Comment:
Don't think its intentional, but it seems you removed some necessary whitespace in this file by accident/automatically

Reply:
Damnit, not again. Xaml Styler is quite aggressive on whitespace. I will move the text to the code behind